index.html

```diff
@@ -633,7 +633,7 @@
 
 <dfn data-lt="governance">Data governance</dfn> is the system of principles that regulate [=information flows=].
 [=Data governance=] determines how
-which [=actors=] can collect what [=data=] and how they may, must, or must not [=process=] it
+ [=actors=] can collect what [=data=] and how they may, must, or must not [=process=] it
```

Comment:
```suggestion
which [=actors=] can collect what [=data=] and how they may, must, or must not [=process=] it
```

That sentence isn't great but we have a better fix in.